jit/v2/JIT_v2.0.0.yaml

```diff
@@ -1181,7 +1195,7 @@ components:
         transportCallReference:
```

Comment:
@HenrikHL, shouldn't this one also be renamed? 
`transportCallReference` -> `terminalCallReference`

Reply:
the assumption is that probably yes. we have kept as it is because it's in use in OVS as transportCallReference, and this would allow to link them. We are thinking that we can still change in in next snapshot and comment that in OVS is presented as transportCallReference. 
Also applies to transportCallSequenceNumber, while not present in OVS, it's used in TNT as transportCallSequenceNumber, but we know in JIT is meant for the sequence of terminal calls within the same port call. 
when I send the Snapshot today I will also send the minute from yesterday weekly call and mention that in our understanding both should be terminalCallReference and terminalCallSequenceNumber, and if any of the carriers oposes to the change in next snapshot.

Reply:
Sounds good Emilio. If would keep it, to match OVS, it is fine as long as it is stated clearly that is refers to OVS and what it means in this context.